rubygem-openscap/rubygem-openscap.spec

```diff
@@ -10,46 +10,46 @@
 
 %global gem_name openscap
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.4.2
-Release: 3%{?dist}
+Name: rubygem-%{gem_name}
+Version: 0.4.3
+Release: 1%{?dist}
 Summary: A FFI wrapper around the OpenSCAP library
 Group: Development/Languages
 License: GPLv2+
-URL: https://github.com/isimluk/ruby-openscap
+URL: https://github.com/OpenSCAP/ruby-openscap
 Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(ffi) >= 1.0.9
+Requires:ruby(rubygems)
+Requires: rubygem(ffi) >= 1.0.9
 # require libopenscap.so.8 in an arch neutral way
 Requires: openscap >= 1.2.1
 Requires: openscap < 1.3.0
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.3
+BuildRequires:rubygems-devel
+BuildRequires:ruby >= 1.9.3
 # For tests we need:
 BuildRequires: openscap >= 1.2.1
 BuildRequires: openscap < 1.3.0
 BuildRequires: bzip2
-BuildRequires: %{?scl_prefix_ruby}rubygem(rake)
-BuildRequires: %{?scl_prefix_ruby}rubygem(bundler)
-BuildRequires: %{?scl_prefix}rubygem(ffi) >= 1.0.9
+BuildRequires: rubygem(rake)
+BuildRequires: rubygem(bundler)
+BuildRequires: rubygem(ffi) >= 1.0.9
 BuildRequires: openscap-devel
 # End (for the tests we needed)
 
 %if 0%{?fedora} > 18
-Requires:      %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
+Requires:     ruby(release)
+BuildRequires: ruby(release)
 # For the tests we need
-BuildRequires: %{?scl_prefix}rubygem(test-unit)
+BuildRequires: rubygem(test-unit)
 %else
-Requires:      %{?scl_prefix_ruby}ruby(abi)
-BuildRequires: %{?scl_prefix_ruby}ruby(abi)
+Requires: ruby(abi)
+BuildRequires: ruby(abi)
 # For the tests we need
-BuildRequires: %{?scl_prefix_ruby}rubygem(minitest)
+BuildRequires:rubygem(minitest)
```

Comment:
Missing space between colon and value, ditto on a few changes above which are misaligned.